### PR TITLE
Store CSVs

### PR DESCRIPTION
**Pull request by nkeyes**
load CSVs, convert to array of hashes, then pass that to store.

also added store_raw

### CODE REVIEW

File: lib/s3mpi/interface.rb [hunk 1]
```diff
@@ -37,10 +38,30 @@ def initialize _bucket, path = ''
     #    The number of times to attempt to store the object.
     def store(obj, key = SecureRandom.uuid, try = 1)
       s3_object(key).write(obj.to_json)
-    rescue 
+    rescue # This should really rescue specific errors
```
Comment (robertzk) [inline lib/s3mpi/interface.rb]: Yes, good point. :) Will think about this later.

File: lib/s3mpi/interface.rb [hunk 2]
```diff
@@ -37,10 +38,30 @@ def initialize _bucket, path = ''
...
+    #    Passed to CSV.read
+    #
+    # @param [Hash] options Options hash.
+    #    Passed to CSV.read
+    def store_csv(csv_file_path, options = Hash.new)
+      store _load_csv(csv_file_path, options)
+    end
+
+    # Store a raw object
+    # @param obj [Object]
```
Comment (robertzk) [inline lib/s3mpi/interface.rb]: Can you use consistent doc formatting? You have type/name above and name/type here.

Comment (nkeyes) [inline lib/s3mpi/interface.rb]: Oops, my bad.

File: spec/s3mpi/interface_spec.rb [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+        {'user_id' => 1, 'email' => 'user1@test.com'},
+        {'user_id' => 2, 'email' => 'user2@test.com'},
+        {'user_id' => 3, 'email' => 'user3@test.com'},
+      ]
+    }
+
+    subject { described_class.new(bucket: bucket, path: path ) }
+
+    describe '.store_csv' do
+      it 'sends the CSV as a row of hashews to .store' do
```
Comment (robertzk) [inline spec/s3mpi/interface_spec.rb]: `s/hashews/hashes` :trollface:

**Comment by robertzk** [comment]
Can you up the [patch version](https://github.com/robertzk/s3mpi-ruby/blob/master/lib/s3mpi/version.rb)?

**Comment by robertzk** [comment]
Thanks!

**Pull request merged**